pkg/apis/acme/v1alpha3/types_issuer.go

```diff
@@ -317,13 +317,13 @@ type ACMEIssuerDNS01ProviderRoute53 struct {
 // ACMEIssuerDNS01ProviderAzureDNS is a structure containing the
 // configuration for Azure DNS
 type ACMEIssuerDNS01ProviderAzureDNS struct {
-	ClientID string `json:"clientID"`
+	ClientID string `json:"clientID,omitempty"`
 
-	ClientSecret cmmeta.SecretKeySelector `json:"clientSecretSecretRef"`
+	ClientSecret *cmmeta.SecretKeySelector `json:"clientSecretSecretRef,omitempty"`
 
 	SubscriptionID string `json:"subscriptionID"`
 
-	TenantID string `json:"tenantID"`
+	TenantID string `json:"tenantID,omitempty"`
```

Comment:
Same comment as `v1alpha2` applies here :) you can copy and paste across the comments

Reply:
done